cg/apps/lims/limsjson.py

```diff
@@ -115,16 +115,17 @@ def expand_case(case_id: str, parsed_case: dict) -> dict:
             gene_panels.update(raw_sample["panels"])
 
         new_sample = {}
+
+        for key, value in raw_sample.items():
```

Comment:
Perhaps its easier to hardcode that we want to create keys fr phenotype terms, synopsis and age at sampling instead? Oherwise maybe they can inject any keys in excel and we parse them

Reply:
I guess you are right, it's a danger.. I guess MMs pydantic PR will mitigate the problem. I think we might change it after MMs PR has been merged in order to ease the burden of solving merge conflicts..

Reply:
👍  if you think that is best and will make us resolve fewer conflicts